TrafficCapture/trafficCaptureProxyServer/src/main/java/org/opensearch/migrations/trafficcapture/proxyserver/netty/NettyScanningHttpProxy.java

```diff
@@ -0,0 +1,62 @@
+package org.opensearch.migrations.trafficcapture.proxyserver.netty;
+
+import io.netty.bootstrap.ServerBootstrap;
+import io.netty.channel.Channel;
+import io.netty.channel.ChannelOption;
+import io.netty.channel.EventLoopGroup;
+import io.netty.channel.nio.NioEventLoopGroup;
+import io.netty.channel.socket.nio.NioServerSocketChannel;
+import io.netty.util.internal.logging.InternalLoggerFactory;
+import io.netty.util.internal.logging.JdkLoggerFactory;
+import org.opensearch.migrations.trafficcapture.IConnectionCaptureFactory;
+
+import javax.net.ssl.SSLEngine;
+import java.util.function.Supplier;
+
+public class NettyScanningHttpProxy {
+    private final int proxyPort;
+    private Channel mainChannel;
+    private EventLoopGroup workerGroup;
+    private EventLoopGroup bossGroup;
+
+    public NettyScanningHttpProxy(int proxyPort) {
+        this.proxyPort = proxyPort;
+    }
+
+    public int getProxyPort() {
+        return proxyPort;
+    }
+
+    public void start(String backsideHost, int backsidePort, Supplier<SSLEngine> sslEngineSupplier,
+                      IConnectionCaptureFactory connectionCaptureFactory)
+            throws InterruptedException {
+        InternalLoggerFactory.setDefaultFactory(JdkLoggerFactory.INSTANCE);
+        bossGroup = new NioEventLoopGroup(1);
+        workerGroup = new NioEventLoopGroup();
+        ServerBootstrap serverBootstrap = new ServerBootstrap();
+        try {
+            mainChannel = serverBootstrap.group(bossGroup, workerGroup)
+                    .channel(NioServerSocketChannel.class)
+                    //.handler(new LoggingHandler(LogLevel.INFO))
+                    //.childHandler(new HexDumpProxyInitializer(backsideHost, backsidePort))
```

Comment:
nit: any reason to keep this commented out code?

Reply:
Second line should be deleted.  First line - judgment call.  When debugging netty, I find myself often having to add logging - so having the pattern "LoggingHandler" that I can search for & being able to uncomment it in about 2 seconds & making sure that it's in the right spot is very helpful.  It's easy to add LoggingHandlers at different places that aren't as helpful - this code would be better if it included a name for the LoggingHandler so that it was really clear.  First we should get comfortable keeping that commented code in place though.